src/components/ReportTransaction.js

```diff
@@ -42,10 +55,19 @@ class ReportTransaction extends Component {
             chatReportID: this.props.chatReportID,
             transactionID: this.props.action.originalMessage.IOUTransactionID,
             comment: '',
+            rejectedIDs: this.props.iouTransactionsBeingRejected,
         });
     }
 
+    isRejected(transactionID, rejectedIDs) {
+        if (!rejectedIDs) {
+            return false;
+        }
+        return rejectedIDs.includes(transactionID.toString());
+    }
+
     render() {
+        const rejectedIDs = lodashGet(this.props, 'iouTransactionsBeingRejected', []);
```

Comment:
Since we're only using `rejectedIDs` in the function `isRejected`, why don't we access the props `this.props.iouTransactionsBeingRejected` and `this.props.action.originalMessage.IOUTransactionID` from within that function? This would probably clean up the `render` method + clean up the conditional rendering below.

Reply:
Yeah, that'd clean up the code.
Done.